framework_tvb/tvb/core/neocom/_h5loader.py

```diff
@@ -161,6 +162,7 @@ def store(self, datatype, fname=None):
             subdt = getattr(datatype, traited_attr.field_name)
             if subdt is not None:  # Because a non required reference may be not populated
                 self.store(subdt)
+        DataEncryptionHandler.push_folder_to_sync(FilesHelper.get_project_folder_from_h5(path))
```

Comment:
is this call (and few others e.g SimAdapter, _h2core.py, h5.py) really necessary, if you add the triggers in hdf5_storage_manager)?